scripts/SpaceCart.js

```diff
@@ -1,3 +1,27 @@
+import { getTransientData, getFacilities, getMinerals, getFacilityInventories } from "./database.js";
+
+const minerals = getMinerals()
+const facilityInv = getFacilityInventories()
+
+document.addEventListener("click", (e) => {
+    if (e.target.name === "facilityInventory") {
+        const spaceCart = {}
+        for (const inv of facilityInv) {
+            if (inv.id === parseInt(e.target.value)) {
+                spaceCart.amount = 1
```

Comment:
If this stays at 1, I don't think we necessarily need to define it as a variable